example/lib/main.dart

```diff
@@ -295,21 +295,27 @@ class _HomePageState extends State<HomePage> {
                   Navigator.push(
                     context,
                     MaterialPageRoute(
-                      builder: (context) => QuestionnaireScroller(
-                        scaffoldBuilder: const CherryBlossomScaffoldBuilder(),
-                        fhirResourceProvider: RegistryFhirResourceProvider([
-                          resourceBundleProvider,
-                          AssetResourceProvider.singleton(
-                            questionnaireResourceUri,
-                            'assets/instruments/sdc_demo.json',
-                          )
-                        ]),
-                        launchContext: launchContext,
-                        questionnaireModelDefaults:
-                            const QuestionnaireModelDefaults(
-                          prefixBuilder: QuestionnaireModelDefaults
-                              .questionNumeralPrefixBuilder,
-                          implicitNullOption: false,
+                      builder: (context) => QuestionnaireTheme(
+                        data: const QuestionnaireThemeData(
+                          horizontalCodingBreakpoint: 100,
+                          allowHorizontalCodingMultipleLines: true,
```

Comment:
@hanny-ph Thanks for the changes. It appears we removed this parameter so the example app is not working now.